src/index.ts

```diff
@@ -65,6 +65,7 @@ function fixDeprecatedOptions(options: any): ProjectOptions {
 export interface ProjectBuildOptions {
   name?: string;
   addServiceWorker?: boolean;
+  addPushManifest?: boolean;
```

Comment:
+1 that would be huge, added new issue to track: https://github.com/Polymer/polymer-project-config/issues/13